src/views/LogsView.js

```diff
@@ -0,0 +1,125 @@
+import React, {useState} from 'react';
+import {graphql, compose} from 'react-apollo';
+import {ALL_EVENTS, MY_PROFILE} from '../state/queries';
+import {
+  Container,
+  Segment,
+  Message,
+  Placeholder,
+  Header,
+  Icon,
+  Select,
+  Button,
+} from 'semantic-ui-react';
+import EmptyView from './EmptyView';
+import EventList from '../components/EventList/EventList';
+import {eventType} from '../common/enums';
+
+const LogsView = ({events: {loading, allEvents, error, refetch}, user}) => {
+  const [typeFilter, setTypeFilter] = useState('');
+
+  const isAdmin = !user.loading
+    ? user.myProfile.roles.includes('ADMIN')
+    : false;
+
+  const eventTypeOptions = Object.keys(eventType).map(type => ({
+    text: eventType[type].title,
+    value: type,
+  }));
+
+  const filterEvents = () => {
+    var filteredEvents = allEvents.edges ? allEvents.edges : [];
+    if (typeFilter) {
+      filteredEvents = filteredEvents.filter(
+        obj => obj.node.eventType === typeFilter,
+      );
+    }
+    return filteredEvents;
+  };
+
+  if (loading)
+    return (
+      <Container as={Segment} basic vertical>
+        <Placeholder>
+          <Placeholder.Header image>
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Header>
+          <Placeholder.Paragraph>
+            <Placeholder.Line />
+            <Placeholder.Line />
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Paragraph>
+        </Placeholder>
+      </Container>
+    );
+  if (error || user.error)
+    return (
+      <Container as={Segment} basic>
+        <Message negative icon>
+          <Icon name="warning circle" />
+          <Message.Content>
+            <Message.Header>Error</Message.Header>
+            <p>Error messages will show up here</p>
+            {error && error.message && <p>Event Error: {error.message}</p>}
+            {user.error && user.error.message && (
+              <p>User Error: {user.error.message}</p>
+            )}
+          </Message.Content>
+        </Message>
+      </Container>
+    );
+  if (isAdmin) {
+    return (
+      <Container as={Segment} basic vertical>
+        <Segment basic floated="right" className="noMargin noPadding">
```

Comment:
Did you consider combining the event list with the existing component in the admin page?